src/compas/geometry/shapes/box.py

```diff
@@ -575,6 +575,28 @@ def scale(self, factor):
         self.ysize *= factor
         self.zsize *= factor
 
+    def scaled(self, x, y=None, z=None):
```

Comment:
why not make `scale` the same and then use it here?

Reply:
yes, of course, you're totally right, changed it here: https://github.com/compas-dev/compas/pull/1399/commits/79596048e0284084a0ec4b360f193d8fb6deaae2

this is now a breaking change because the argument names have changed, even if positionally it would be a non-breaking change